pyfr/writers/paraview.py

```diff
@@ -290,6 +361,8 @@ def _write_vtu_header(args, vtuf, m_inf, s_inf, off):
 
     write_s = lambda s: vtuf.write(s.encode('utf-8'))
 
+    pp = (lambda x: 'P'+x) if parallel else (lambda x: x)
```

Comment:
Consider defining a variable like `prefix = 'P' if parallel else ''` and then just using that where needed: 'stuff...{prefix}...'.format(..., prefix=prefix)`, also avoids the use of '+' for string building.